algorand/go-algorand
# onetechnical/relbeta2.1.6

## Summary

## Highlights

A bug in the algorand debian package would result in the genesis.json for TestNet being applied instead of MainNet. Debian users should verify that `/var/lib/algorand/genesis.json` is pointing to the correct network after upgrading. **This is especially true for TestNet users who upgraded to the 2.1.5 debian package.**

## Changes

1. Other
    * Bug Fix - use DEFAULT_RELEASE_NETWORK genesis file in debian package

## Test Plan

Standard release testing.

For a rel/stable release, do the following tests:

MainNet w/ 2.1.5 TestNet downgrade:
- Install 2.1.4 deb, verify /var/lib/algorand/genesis.json is pointing to mainnet
- Install 2.1.5 deb, verify /var/lib/algorand/genesis.json is pointing to testnet (replicating bug)
- Install 2.1.6 deb, verify /var/lib/algorand/genesis.json is pointing to mainnet

MainNet 2.1.6 install:
- Install 2.1.6 deb, verify /var/lib/algorand/genesis.json is pointing to mainnet

MainNet 2.1.5 install (config changed) upgraded to 2.1.6:
- Install 2.1.5 deb, verify /var/lib/algorand/genesis.json is pointing to testnet
- Modify /var/lib/algorand/genesis.json to point to mainnet's genesis.json
- Install 2.1.6 deb, verify that /var/lib/algorand/genesis.json continues to point to mainnet
